vars.cmd

```diff
@@ -10,12 +10,17 @@ set JENKINS_JAVA_OPTS=-Xmx1280m
 @REM Jenkins binds to 0.0.0.0 (OMG) so we can use any network but the docker network.
 @REM but we may as well use the default network
 @echo off
-FOR /F "tokens=* USEBACKQ" %%F IN (`netsh interface ipv4 show addresses "vEthernet (WSL)" ^| grep "IP Address:" ^| gawk '{print $3}'`) DO (
+FOR /f "tokens=3" %%F in ('netsh interface ipv4 show addresses "vEthernet (WSL)" ^| findstr /c:"IP Address:"') DO (
 SET IP=%%F
 )
+IF NOT DEFINED IP (

```

Comment:
because I always forget to start docker first - make it obvious to myself (and others!)